src/garage/torch/algos/loss_function_utils.py

```diff
@@ -1,10 +1,25 @@
-"""loss function utilities."""
+"""Differentiable utils typically used when computing loss functions."""
 import torch
 import torch.nn.functional as F
 
 
 def compute_advantages(discount, gae_lambda, max_len, baselines, rewards):
-    """Calculate advantages."""
+    """
+    Calculate advantages.
```

Comment:
"Calculate advantages using a baseline (value function) according to Generalized Advantage Estimation (GAE)"

Reply:
fixed